### PR TITLE
implemented inserts to fstab via UUID:

### PR DESCRIPTION
**Pull request by ezjust**
- made correction into $DISK variable value - now it could be set without case sensitivity 
- added new type of notes those will be write to fstab if --format=uuid option is set 
- added decryption of "format","version","help" options

### CODE REVIEW

**Comment by ezjust** [comment]
-made changes into fstab function, merged two functions into single.
- set variables case $FORMAT, $DISK into menu list.

File: configurator.sh [hunk 1]
```diff
@@ -108,7 +114,7 @@ if [[ -z "${disks[0]}" || -z "${disks[1]}" || -z "${disks[2]}" || -z "${disks[3]
         echo "${disks[1]}"
         echo "${disks[2]}"
         echo "${disks[3]}"
-        echo "${disks[4]}"
+	        echo "${disks[4]}"
```
Comment (MBugaiov) [inline configurator.sh]: Here is missmatch

File: configurator.sh [hunk 2]
```diff
@@ -812,24 +818,27 @@ function extended {
...
+	 if [[ $FORMAT = "uuid" ]]; then
+	    exp=$(echo ${fstab[$i]} | awk -F " " '{print $1}')
+            uuid=$(blkid -o export $exp | grep "^UUID=") 
+	    mpfs=$(echo ${fstab[$i]} | awk -F " " '{print $2,$3}') 
+	    echo $uuid $mpfs defaults 0 0 >> /etc/fstab
+	 else
+	    if [[ "${fstab[$i]}" == "/dev/md124p1 /mnt/md5p1 ext4" ]]; then
+            fstab[$i]="/dev/md/md5p1 /mnt/md5p1 ext4" # since after reboot /dev/md124p1 becames /dev/md/md5p1 we use check for this device during mounting and use /dev/md/md5p1 as a default path for this device
+            fi
+	    echo ${fstab[$i]} defaults 0 0 >> /etc/fstab
```
Comment (MBugaiov) [inline configurator.sh]: Please, check spaces

File: configurator.sh [hunk 3]
```diff
@@ -812,24 +818,27 @@ function extended {
...
-mount -a
-}
+     function fstab {
+     IFS_OLD=$IFS
+     IFS=$'\n'
+     set -o noglob
+     fstab=( `cat /proc/mounts | grep '_ext2\|_ext3\|_ext4\|_xfs\|_btrfs\|-linear_0\|-stripe_0\|_separate\|-mirror_0\|partition-ext4\|md5p1\|thinlvm\|md-mirror-md-lvm\|md127p2-' | awk '{print $1,$2,$3}'` )
+     	for ((i = 0; i < ${#fstab[@]}; i++)); do
+	 if [[ $FORMAT = "uuid" ]]; then
+	    exp=$(echo ${fstab[$i]} | awk -F " " '{print $1}')
```
Comment (MBugaiov) [inline configurator.sh]: awk -F " " '{print $1}'   is similar to awk '{print $1}'

**Comment by MBugaiov** [comment]
+1:

**Pull request merged**